content/docs/dvclive/dvclive-with-dvc.md

```diff
@@ -38,22 +38,20 @@ for epoch in range(NUM_EPOCHS):
     live.next_step()
 ```
 
-Let's use `dvc stage add` to create a stage to wrap this code (don't forget to
+Let's use `dvc exp init` to create a stage to wrap this code (don't forget to
```

Comment:
question to double check - is `dvc init` still needed?

Reply:
According to the [`exp init` ref](https://dvc.org/doc/command-reference/exp/init), `init` is always needed first.

Reply:
It is still needed. Running `exp init` without `dvc init` will raise `NotDvcRepoError`